src/popup/router/components/AccountItem.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div
+    class="account-item"
+    :class="{ 'has-name': name }"
+  >
+    <div
+      v-if="name"
+      class="head"
+    >
+      <Avatar
+        :address="address"
+        :name="name"
+      />
+      <span class="name">{{ name }}</span>
+    </div>
+    <Avatar
+      v-else
+      :address="address"
+    />
+    <span class="address">{{ address }}</span>
+  </div>
+</template>
+
+<script>
+import Avatar from './Avatar';
+
+export default {
+  components: { Avatar },
+  props: {
+    address: { type: String, required: true },
+    name: { type: String, default: '' },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@use '../../../styles/variables';
+@use '../../../styles/typography';
+
+.account-item {
+  display: flex;
+  align-items: center;
+
+  .avatar {
+    width: 32px;
+    height: 32px;
```

Comment:
```suggestion
  .avatar {
    width: 32px;
    height: 32px;
```
Maybe we shall introduce new Avatar size, due to usage in several places.
here,
SidebarMenu,
New Names page designs.